File: atcoder/codefest2015final/a.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
)

var _ = fmt.Scanf
var sc = bufio.NewScanner(os.Stdin)

func nextInt() int {
	sc.Scan()
	i, e := strconv.Atoi(sc.Text())
	if e != nil {
		panic(e)
	}
	return i
}

func main() {
     sc.Split(bufio.ScanWords)
     sc.Scan(); a := sc.Text()
     sc.Scan(); b := sc.Text()
     sc.Scan(); c := sc.Text()
     if len(a) == 5 && len(b) == 7 && len(c) == 5 {
        fmt.Println("valid")
     } else {
	fmt.Println("invalid")
    }  
}		

File: atcoder/codefest2015final/a.cs
using System.Linq;
using System;

public class A {
	public static void Main(string[] args) {
		int[] val = Console.ReadLine().Split(new []{' '}).Select(x => x.Length).ToArray();
		Console.WriteLine(val[0] == 5 && val[1] == 7 && val[2] == 5 ? "valid" : "invalid");
	}
}